jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -271,12 +324,91 @@ public void callback(long args) {
             glfwSwapInterval(0);
         }
 
+        final Object[] icons = settings.getIcons();
+
+        if (icons != null) {
+
+            final GLFWImage[] images = imagesToGLFWImages(icons);
+            final GLFWImage.Buffer iconSet = GLFWImage.malloc(images.length);
+
+            for (int i = images.length - 1; i >= 0; i--) {
```

Comment:
Can just use a for-each loop here, e.g.:
```
for (GLFWImage image : images)
```
Unless the image ordering matters somehow. Based on the GLFW documentation, it doesn't seem like it does.